recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
+  version: "{{ version }}"
+
+source:
+  fn: hdfs3-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/h/hdfs3/hdfs3-{{ version }}.tar.gz
+  md5: ed0810314af1f4c147af0c1d6ed8a266
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  # This packages uses libhdfs3, which is primarily used with HDFS on Linux.
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - libhdfs3
+
+  run:
+    - python
+    - libhdfs3
+
+test:
+  imports:
+    - hdfs3
+
+about:
+  home: https://github.com/dask/hdfs3
+  license: BSD 3-Clause
+  summary: Python wrapper for libhdfs3
+
+extra:
+  recipe-maintainers:
+    - mrocklin
+    - martindurant
```

Comment:
Yes. In general I am fine being listed as a maintainer under any project
that Kris Overholt lists me as such or for any project listed as being
under the dask github organization.
On May 21, 2016 5:01 PM, "Martin Durant" <notifications@github.com> wrote:

> In recipes/hdfs3/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/635#discussion_r64140217>
> :
>
> > +    - python
> > +    - libhdfs3
> > +
> > +test:
> > +  imports:
> > +    - hdfs3
> > +
> > +about:
> > +  home: https://github.com/dask/hdfs3
> > +  license: BSD 3-Clause
> > +  summary: Python wrapper for libhdfs3
> > +
> > +extra:
> > +  recipe-maintainers:
> > +    - mrocklin
> > +    - martindurant
>
> OK with me
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/635/files/81ea3c59121f5f0c9e1920764007e8c9f2cda917#r64140217>
>